Dockerfile.adm-webhook

```diff
@@ -0,0 +1,20 @@
+# Build stage
+ARG goversion
+FROM golang:${goversion}-alpine as builder
+RUN apk add build-base git mercurial
+# Download modules in a separate step for quicker builds when deps haven't changed
+ADD go.mod /spire/go.mod
+ADD go.sum /spire/go.sum
+RUN cd /spire && go mod download
+# Build admission webhook
+ADD . /spire
+RUN cd /spire/support/k8s/spire-adm-webhook && go test && go build
+
+# Image stage
+FROM alpine
+RUN apk add dumb-init ca-certificates
```

Comment:
Not really since we're not doing anything with Web PKI.... I'll remove it and also alter the other Dockerfile's to do the `apk add` on separate lines so we can benefit from intermediate containers.